league-content/web-mobile.md

```diff
@@ -5,10 +5,15 @@ Resources of interest to students and faculty all about web and mobile developme
 ## Talks
 
 ## Code Samples
-
+[A simple clone of the game Wordle using React and tailwind CSS]()
```

Comment:
hi, if this is something you've created, can you link it here? looks interesting!

Reply:
I am sorry, I forgot to add the link in the last commit, Now I added it

Reply:
Wow I love this! Yes, let's include it. In fact I'd love for you to deploy this as a Static Web App and write a blogpost about it, how can I help make that happen?

Reply:
I would love to do this, Hosting the app as a static web app won't be an issue since every chance to work with azure is a delight for me ☺️, but I can use some help on the structure of the blog since I don't have experience writing blogs 😬

Also, the repository was shared on this website: https://reactjsexample.com/wordle-xd-a-worlde-game-made-using-reactjs-and-tailwind-css/